test/integration/coscheduling_test.go

```diff
@@ -370,17 +370,18 @@ func TestCoschedulingPlugin(t *testing.T) {
 
 			}
 			// A workaround to speed up the test by moving unschedulable Pods back to activeQ.
-			wait.PollUntil(time.Millisecond*200, func() (bool, error) {
-				numUnschedulablePods := testCtx.Scheduler.SchedulingQueue.NumUnschedulablePods()
-				return numUnschedulablePods > 0, nil
-			}, testCtx.Ctx.Done())
-			klog.Infof("Start to move unschedulable Pods back to activeQ.")
-			// DEBUG
-			for _, p := range testCtx.Scheduler.SchedulingQueue.PendingPods() {
-				klog.Infof("[DEBUG] pod %v is pending", p.Name)
-			}
-			testCtx.Scheduler.SchedulingQueue.MoveAllToActiveOrBackoffQueue("fake event")
-
+			go func() {
```

Comment:
Maybe we should remove this entirely. I tested it locally by removing the whole go routine, all tests can be finished within 3 mins, which I think is ok. WDYT?

```
⇒  cat /tmp/coscheduling-test-*.log
ok  	sigs.k8s.io/scheduler-plugins/test/integration	188.320s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	98.227s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	97.789s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.961s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.698s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.755s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.520s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	103.624s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	188.268s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	98.292s
```

Reply:
It seems keep this logic still help speed up the tests a lot. With this logic, most tests would finish in 2s

Reply:
If the test still passes after removing this logic, I think we can remove it to keep it consistent with the scheduler process. Previously, this logic was added to avoid timeout, but I would like to know why there is no timeout after removing this logic now?
@Huang-Wei @cwdsuzhou

Reply:
Another 10 runs have the same result:

```
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.840s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	97.789s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.849s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.961s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.698s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.755s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.520s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	103.624s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	188.268s
ok  	sigs.k8s.io/scheduler-plugins/test/integration	98.292s
```

Reply:
> Another 10 runs have the same result:
> 
> ```
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.840s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	97.789s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.849s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.961s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.698s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.755s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	187.520s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	103.624s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	188.268s
> ok  	sigs.k8s.io/scheduler-plugins/test/integration	98.292s
> ```


Yes, seems we can remove it now, stats in my environment are also similar, details as follow:

Move to goroutine
    --- PASS: TestCoschedulingPlugin/equal_priority,_sequentially_pg1_meet_min_and_pg2_not_meet_min (1.49s)
    --- PASS: TestCoschedulingPlugin/equal_priority,_not_sequentially_pg1_meet_min_and_pg2_not_meet_min (89.42s)
    --- PASS: TestCoschedulingPlugin/equal_priority,_not_sequentially_pg1_not_meet_min_and_3_regular_pods (1.28s)
    --- PASS: TestCoschedulingPlugin/different_priority,_sequentially_pg1_meet_min_and_pg2_meet_min (1.29s)
    --- PASS: TestCoschedulingPlugin/different_priority,_not_sequentially_pg1_meet_min_and_pg2_meet_min (1.32s)
    --- PASS: TestCoschedulingPlugin/different_priority,_not_sequentially_pg1_meet_min_and_3_regular_pods (1.22s)
    --- PASS: TestCoschedulingPlugin/equal_priority,_not_sequentially_pg1_meet_min_and_p2_p3_not_meet_min (84.55s)

Remove this logic:
    --- PASS: TestCoschedulingPlugin/equal_priority,_sequentially_pg1_meet_min_and_pg2_not_meet_min (1.39s)
    --- PASS: TestCoschedulingPlugin/equal_priority,_not_sequentially_pg1_meet_min_and_pg2_not_meet_min (89.39s)
    --- PASS: TestCoschedulingPlugin/equal_priority,_not_sequentially_pg1_not_meet_min_and_3_regular_pods (1.26s)
    --- PASS: TestCoschedulingPlugin/different_priority,_sequentially_pg1_meet_min_and_pg2_meet_min (1.29s)
    --- PASS: TestCoschedulingPlugin/different_priority,_not_sequentially_pg1_meet_min_and_pg2_meet_min (1.28s)
    --- PASS: TestCoschedulingPlugin/different_priority,_not_sequentially_pg1_meet_min_and_3_regular_pods (1.24s)
    --- PASS: TestCoschedulingPlugin/equal_priority,_not_sequentially_pg1_meet_min_and_p2_p3_not_meet_min (85.55s)

@deads2k  timeout is 120s for every case, current implementation would not reach 120s


Would update this PR and remove this logic